core/src/main/scala/kafka/server/ReplicaFetcherBlockingSend.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.kafka.common.Node
 import org.apache.kafka.common.requests.AbstractRequest.Builder
 
 import scala.collection.JavaConverters._
-import org.apache.kafka.common.config.ClientDnsLookup
+import org.apache.kafka.clients.ClientDnsLookup
```

Comment:
There is a wildcard import in this class for org.apache.kafka.clients, so this import is not required.